color_r: 1.0
color_g: 0.4
color_b: 0.0
bbox_min_x: 0.0
bbox_min_y: 0.0
bbox_min_z: 0.0
bbox_max_x: 48.0
bbox_max_y: 53.0
bbox_max_z: 6.0
width: 0.3
alpha: 1.0